service-13/meta.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "ST[13] large packet transfer",
+  "submodules": [
+    {
+      "name": "Large packet downlink subservice",
+      "elements": [
+        {
+          "name": "Large packet downlink reports",
+          "asn1Files": [
+            "PUS-13-1.asn1",
+            "PUS-13-2.asn1",
+            "PUS-13-3.asn1",
+            "LargeMessageTransaction.asn1"
+          ],
+          "imports": [
+            {
+              "from": "BasicTypes",
+              "types": "PUSC-UINT32"
```

Comment:
Types should be an array. It might be irrelevant, as this field isn't probably needed, but keep library's convention for now